src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -41,10 +41,6 @@ def path_lookup(self):
     def dependency(self):
         return self._dependency
 
-    @property
```

Comment:
How does this relate to this PR ? Every node in the graph points to a path, so not sure this is a simplification ?

Reply:
I removed this in 5993eea3df109249fb162f5781079a7dd74b3992, because our linting rules only allow for 20 public methods and: a) we had 20; b) another arrives in the next commit.

It's either remove an existing method, or add a linting suppression. I chose to remove this because it wasn't used externally and was trivial in nature.

Reply:
eh... i'd rather have it done in a separate PR